lib/situation_room/site_checks/site_check_controller.ex

```diff
@@ -0,0 +1,28 @@
+defmodule SituationRoom.Site.Check.Controller do
+  @moduledoc """
+  Documentation for `SituationRoom.Site.Controller`.
+  """
+  use Plug.Router
+  alias SituationRoom.Site.Check
+
+  plug(:match)
+
+  plug(Plug.Parsers,
+    parsers: [:urlencoded, :multipart],
+    pass: ["text/*"]
+  )
```

Comment:
What are these parsers here for?  We don't need any parsing as we only have a GET endpoint?